docs/usb/wdi-usb-interface.md

```diff
@@ -33,13 +38,29 @@ https://gitlab.freedesktop.org/libevdev/libevdev/-/blob/master/include/linux/lin
  BTN_THUMBR
 ```
 
-## Drive actions
+## Drive Actions
 | Action | Keyboard Controls | Gamepad Controls |
 |--------|------|-----|
 | Drive forward | 'w' \| up arrow | Y axis (towards min val) |
 | Drive backward | 's' \| down arrow | Y axis (towards max val) |
 | Drive left | 'a' \| left arrow | X axis (towards min val) |
 | Drive right | 'd' \| right arrow | X axis (towards max val) |
-| Emergency stop | backspace | button_east?? would like this to be something that is easy to press under stress |
-| Enable drive | enter | start button |
-| 
\ No newline at end of file
+| Emergency stop | backspace | BTN_SOUTH \| BTN_THUMBL?? |
```

Comment:
`BTN_THUMBL` is the button when the left analog stick is pressed down on gamepads. My thinking is that in moments of panic someone may tense up and press this, which would E-stop; however, I could also see this sort of thing happening when someone is going across the street, which would cause more anxiety and be less safe. Any thoughts?

Reply:
I recommend we drop the BTN_THUMBL here